amqphandler/amqphandler_test.go

```diff
@@ -320,62 +329,250 @@ func TestSendMessages(t *testing.T) {
 	}
 }
 
-func TestReceiveMessages(t *testing.T) {
-	systemID := "testID"
+func TestDesiredStatusMessages(t *testing.T) {
+	var (
+		sendDesiredStatus = cloudprotocol.DesiredStatus{
+			BoardConfig:  []byte("error"),
```

Comment:
Why `error` ? It sounds negative.

Reply:
just to mark that we have incorrect desired status element